content/en/api/dashboard_lists/code_snippets/result.api-dashboard-list-get-items.py

```diff
@@ -46,10 +49,11 @@
             'is_favorite': False,
             'is_read_only': False,
             'type': 'custom_timeboard',
-            'id': 757575,
+            'id': 'jk9-p5y-ii6',
```

Comment:
💯 for this field, but we also need to use the new`url` format for `custom_timeboard` and `custom_screenboard`. In this case, the `url` field is `/dashboard/jk9-p5y-ii6/trace-api `. Same for screenboard samples.